odl/tomo/geometry/parallel.py

```diff
@@ -426,6 +427,63 @@ def __repr__(self):
     rotation_matrix = AxisOrientedGeometry.rotation_matrix
 
 
+def parallel_beam_geometry(space, angles=None, det_shape=None):
+    """Create default parallel beam geometry from space.
+
+    This default geometry gives a fully sampled sinogram according to the
+    nyquist criterium. In general this gives a very large number of samples.
+
+    This is intended for simple test cases where users do not need the full
+    flexibility of the geometries, but simply wants a geomoetry that works.
+
+    Parameters
+    ----------
+    space : `DiscreteLp`
+        Space in which the image should live. Needs to be 2d or 3d.
+    angles : int, optional
+        Number of angles. Default: Enough to fully sample the data.
```

Comment:
"fully sample the data" - I get what you mean, but it could be clearer. Why not refer to the Nyquist criterion as you did before and perhaps include a link to a source above? Like [this one](http://wwwmath.uni-muenster.de/num/Preprints/1998/natterer_1/paper.pdf).